.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
```

Comment:
- The removal of extra HTML may affect the formatting of the content, ensure it doesn't remove essential elements needed for display.